docs/Using-Fleet/configuration-files/README.md

```diff
@@ -509,7 +509,7 @@ spec:
 WARNING: this will log potentially a lot of data. Some of that data might be private, please verify it before posting it
```

Comment:
```suggestion
> **Warning:** this will potentially log a lot of data. Some of that data might be private, please verify it before posting it
```
Here too. Along with a grammar mishap.